tests/command.bats

```diff
@@ -381,4 +381,24 @@ setup() {
 
   unstub mktemp
   unstub aws
-}
\ No newline at end of file
+}
+
+
+@test "Post-command skips with flag" {
+
+  export BUILDKITE_PLUGIN_CACHE_SKIP_POST_COMMAND_HOOK="true"
+
+  run "$PWD/hooks/post-command"
+  assert_success
+  refute_output --partial ':bash: Processing'
+  assert_output --partial 'SKIP_POST_COMMAND_HOOK flag Detected. Skipping...'
+
+  unset BUILDKITE_PLUGIN_CACHE_SKIP_POST_COMMAND_HOOK
+}
+
+@test "Post-command is not skipped by default" {
+
+  run "$PWD/hooks/post-command"
+  assert_output --partial ':bash: Processing'
```

Comment:
for safety, would it make sense to add a refute that it's not skipping?

refute_output --partial 'SKIP_POST_COMMAND_HOOK flag Detected. Skipping...'

Reply:
added